primitives/arkworks/Cargo.toml

```diff
@@ -0,0 +1,59 @@
+[package]
+name = "sp-arkworks"
+version = "0.4.0"
+authors = ["Parity Technologies <admin@parity.io>"]
+edition = "2021"
+license = "Apache-2.0"
+homepage = "https://substrate.io"
+repository = "https://github.com/paritytech/substrate/"
+
+[package.metadata.docs.rs]
+targets = ["x86_64-unknown-linux-gnu"]
+
+[dependencies]
+ark-serialize = { version = "0.4.0", default-features = false }
+ark-ff = { version = "0.4.0", default-features = false }
+ark-ec = { version = "0.4.0", default-features = false }
+ark-std = { version = "0.4.0", default-features = false }
+ark-bls12-377 = { version = "0.4.0", features = ["curve"], default-features = false }
+ark-bls12-381 = { version = "0.4.0", features = ["curve"], default-features = false }
+ark-bw6-761 = { version = "0.4.0", default-features = false }
+ark-ed-on-bls12-381-bandersnatch = { version = "0.4.0", default-features = false }
+ark-ed-on-bls12-377 = { version = "0.4.0", default-features = false }
+sp-std = { version = "5.0.0", path = "../std", default-features = false }
+codec = { package = "parity-scale-codec", version = "3.2.2", default-features = false }
+ark-scale = { git = "https://github.com/w3f/ark-scale", features = ["hazmat"], default-features = false }
```

Comment:
git dependencies are not allowed in substrate as far as I know. Crates from https://github.com/w3f/ark-scale and  https://github.com/paritytech/ark-substrate will have to be released on crates.io
@bkchr Please correct if I'm wrong here.

Reply:
I got everything published to crates.io and removed the git dependencies!